cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -221,6 +225,10 @@
     private KafkaAuthorization authorization;
     private KafkaVersion kafkaVersion;
     private CruiseControlSpec cruiseControlSpec;
+    private String kafkaDefaultNumPartitions = "1";
+    private String kafkaDefaultReplicationFactor = "1";
+    private String ccDefaultNumPartitions = null;
+    private String ccDefaultReplicationFactor = null;
```

Comment:
I think the names here are confusing. If I read the code right ... the `kafkaDefault*` are the defaults which will be used when user doesn't specify anything specific size? And the `ccDefault*` are the actual value either user configured of default? In which case they should not be named `default` because that is confusing and when you see default you expect there to be some other non-default value. So IMHO you should rename it (including the KafkaBrokerConfiguration etc.)